packages/components/src/alignment-matrix-control/index.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import classnames from 'classnames';
+import {
+	unstable_useCompositeState as useCompositeState,
+	unstable_Composite as Composite,
+	unstable_CompositeGroup as CompositeGroup,
+} from 'reakit';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useInstanceId } from '@wordpress/compose';
+import { useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+import { Root, Row } from './styles/alignment-matrix-control-styles';
+import { useRTL } from '../utils/rtl';
+import AlignmentMatrixControlIcon from './icon';
+import { GRID, getItemId } from './utils';
+
+function useBaseId( id ) {
```

Comment:
Do we really need this customized id? Why not just use the regular `useInstanceId`. 
Alternatively, we could support prefixes in `useInstanceId` directly

Reply:
> Alternatively, we could support prefixes in useInstanceId directly

I think that may be a good idea! It looks like manually prefixing `instanceId` seems to be the common pattern, example:

```js
	const instanceId = useInstanceId( AnglePickerControl );
	const inputId = `components-angle-picker-control__input-${ instanceId }`;
```